src/components/NotificationRow.tsx

```diff
@@ -108,7 +108,7 @@ export const NotificationRow: FC<IProps> = ({ notification }) => {
         className={cn('mr-3 flex w-5 items-center justify-center', iconColor)}
         title={notificationTitle}
       >
-        <NotificationIcon size={18} aria-label={notification.subject.type} />
+        <NotificationIcon size={14} aria-label={notification.subject.type} />
```

Comment:
```suggestion
        <NotificationIcon size={16} aria-label={notification.subject.type} />
```
@setchy let's agree on this middleground?